src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -516,6 +536,15 @@ object GardenVisitorFeatures {
                 ChatUtils.chat("Talk to the visitor again to update the number of items needed!")
             }
         }
+
+        if (GardenAPI.inGarden() && config.compactVisitorRewardChat && (
```

Comment:
Is there a reason why we use events from GardenVisitorFeatures for compact chat? Can we please move the event and all regex patterns into GardenVisitorCompactChat class?